packages/gatsby/src/schema/build-connection-fields.js

```diff
@@ -104,7 +104,30 @@ module.exports = type => {
           groupConnections.push(groupConn)
         })
 
-        return groupConnections
+        // make tags arrays case ignored
+        const tags = groupConnections.map(e=>{
```

Comment:
Yeah, my code was kinda messy definitely needs improvement. I was thinking to make an option for the `group` in `Graphql` query. something like this
```
    allMarkdownRemark {
      group(field: frontmatter___tags, caseSensitive: no) {
        fieldValue
        totalCount
      }
```